components/message-toolbar.jsx

```diff
@@ -28,11 +28,20 @@ export default class MessageToolbar extends PureComponent {
         this.props.replyTo(this.props.mxEvent);
     }
     
+    /** Quote multiline messages */
+    generateQuoted(msg) {
```

Comment:
there's a really tidy (and simple) regex that can replace this function

Reply:
Yeah, my bad. I'll replace it.